pkgs/applications/window-managers/leftwm/default.nix

```diff
@@ -20,7 +21,22 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-wn5DurPWFwSUtc5naEL4lBSQpKWTJkugpN9mKx+Ed2Y=";
   };
 
-  cargoHash = "sha256-TylRxdpAVuGtZ3Lm8je6FZ0JUwetBi6mOGRoT2M3Jyk=";
+  cargoPatches = [
+    # This pacth can be removed with the next version bump, it just updates the `time` crate
+    (fetchpatch {
+      url = "https://github.com/VuiMuich/leftwm/commit/9358bebb11df19f46d0813723959518498d812b2.patch";
+      hash = "sha256-MP74lSmH+dR903dzZ/e6nsHSBpK9mbvOFUlrK6//+38=";
+    })
+  ];
+
+  # To show the "correct" git-hash in `leftwm-check` we manually replace the respective string
+  # can be remove together with the above patch
+  postPatch = ''
+    substituteInPlace leftwm/src/bin/leftwm-check.rs \
+      --replace-fail 'option_env!("GIT_HASH").unwrap_or(git_version::git_version!(fallback = "unknown"))' '"36609e0 patched with `VuiMuich/leftwm: 9358beb`"'
```

Comment:
Why not set the environment variable it's explicitly checking for rather than patching?

Reply:
Well sometimes you miss to see the obvious solution :angel: